docs/en/serial-vault.md

```diff
@@ -0,0 +1,415 @@
+signed and issued.
+
+The Signing Log displays the list of serial numbers and device keys that have
+been used to sign serial assertions. You can also filter the log. 
+
+![image][serial-vault-image8]
+
+## Creating system-user assertions
+
+A system-user assertion allows you to create a user on an unmanaged Core
+system. The system-user assertion can be seeded into an image, or it can be
+auto-imported from a USB drive. See here and [system user][system-user] and
+[system user assertions.][system-user-assertions.] 
+
+The system-user assertion is signed by a key you create, register, export and
+upload to the Serial Vault just like the serial key. See the the content
+elsewhere in this doc for detailed instructions.
+ 
+The System-User menu lets you create system user assertion for a certain brand
+/model, as follows:
+1. Click System-User.
+1. Enter the email of the system user into Email field.
+1. Enter the desired login name of the system user in Username field.
+1. Enter the system user password in Password field.
+1. Enter complete name of the user in Full Name field.
+1. Select the model of the device this user is being created from Model combo
+box.
+1. Set date and time in UTC since this assertion is valid in Since (UTC)
+field.
+
+!!! Note:
+          This date must be after the date on which the key used to sign the
+          system user assertion was registered.  
+1. Press Create to generate the assertion.
+
+!['image'][serial-vault-image9]
+
+After pressing Create, the generated assertion is displayed. Click Download 
+to save it to a local file. 
+
+As explained in linked documents, you can place this file in the root
+directory of a USB drive and named auto-import.assert. If the USB drive is
+inserted in an unmanaged Core system, the assertion is imported and the system
+user is created. 
+
+## Verifying a system obtained a serial assertion
+
+You can verify from a terminal session on a system if a serial assertion has
+been retrieved from the Serial Vault in various ways.
+
+
+!!! Note:
+          If the Core system does not have a user so you cannot get a terminal
+          session into the device, you can create one with a system-user
+          assertion as explained elsewhere in this document.
+
+### Verifying serial with snap known serial
+
+Display the system’s serial assertion, if any, as follows:
+
+```bash
+snap known serial
+```
+
+The output should show a variety of information, including the serial identifier:
+
+```bash
+type: serial
+authority-id: canonical
+brand-id: mybrand
+model: rpi3
+serial: 381d1f99-749f-41f5-a667-608f698a4a4a
+device-key:
+	AcbBTQRWhcGAARAAzFqvP/235lVcpelVL2RxxHnDwi0viTLFdk3QxrSDfmLlTrP/griNH5zfDPHH
+Qf7mCC2yrjPLgSPmDvZm58Mnh3y+TFL9dF6lF6L0qXM865LO07sD1nZKE9lvl61jt+opmHviwTvb
+...
+```
+
+If there is no output from this command, the system does not have a serial assertion.
+
+
+
+### Checking Initialize device transaction with snap changes
```

Comment:
Shouldn't "Initialize" be lowercase?

Reply:
I want to see what David and Nick think here. As you can see in the proposed doc[1], the term "Initialize device" is in the Summary column of the cli output, so it is a quote of GUI. Really though, this is also the *type* of transaction, it is a nameable transaction that every core system tries to complete. 

[1] https://github.com/evilnick/ubuntu-core-docs/blob/serial-vault/docs/en/serial-vault.md

Reply:
If it reflects the UI, I'd keep it like this

Reply:
if it has some extra meaning it should be identified somehow. usually we put such things in the inline code style, which is what I will do here so it becomes similar to:

## Checking `Initialize device` transaction with snap changes